src/assets/icons/offline.svg

```diff
@@ -0,0 +1,14 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+  <path d="M13.1,19.1c0.6-0.6,0.6-1.6,0-2.3c-0.6-0.6-1.7-0.6-2.3,0c-0.6,0.6-0.6,1.6,0,2.3C11.5,19.7,12.5,19.7,13.1,19.1z
+    "/>
+  <path d="M16.8,14.8l-0.5-0.5c-2.3-2.3-6.1-2.3-8.5,0l-0.5,0.5C7,15,7,15.4,7.2,15.7l0.5,0.5c0.1,0.1,0.3,0.2,0.4,0.2
+    c0.2,0,0.3-0.1,0.4-0.2l0.5-0.5c1.5-1.5,4.1-1.5,5.6,0l0.5,0.5c0.2,0.2,0.6,0.2,0.9,0l0.5-0.5c0.1-0.1,0.2-0.3,0.2-0.4
+    C16.9,15.1,16.9,14.9,16.8,14.8z"/>
+  <path d="M18.9,12.1l-0.5-0.5c-3.5-3.5-9.2-3.5-12.7,0l-0.5,0.5c-0.1,0.1-0.2,0.3-0.2,0.4c0,0.2,0.1,0.3,0.2,0.4l0.5,0.5
+    c0.2,0.2,0.7,0.2,0.9,0L7.1,13c1.3-1.3,3.1-2,4.9-2s3.6,0.7,4.9,2l0.5,0.5c0.1,0.1,0.3,0.2,0.4,0.2c0,0,0,0,0,0
+    c0.2,0,0.3-0.1,0.4-0.2l0.5-0.5C19.1,12.8,19.1,12.4,18.9,12.1z"/>
+  <path d="M21,9.5L20.5,9c-4.7-4.7-12.3-4.7-17,0L3,9.5C2.8,9.6,2.8,9.8,2.8,9.9c0,0.2,0.1,0.3,0.2,0.4l0.5,0.5c0.2,0.2,0.6,0.2,0.9,0
+    l0.5-0.5c3.9-3.9,10.3-3.9,14.2,0l0.5,0.5c0.1,0.1,0.3,0.2,0.4,0.2s0.3-0.1,0.4-0.2l0.5-0.5C21.3,10.1,21.3,9.7,21,9.5z"/>
+  <path d="M12,23c6.1,0,11-4.9,11-11S18.1,1,12,1S1,5.9,1,12S5.9,23,12,23z M19.8,5.1l-16,13.7" fill="none" stroke-width="2"/>
+</svg>
+
```

Comment:
This icon feels like it could be more consistent with other elements of our patterns.

I think this is especially clear if you compare it to the way our `bell` icon looks inside of a slashed button:

<img width="161" alt="Screen Shot 2021-11-16 at 2 10 06 PM" src="https://user-images.githubusercontent.com/69633/142074200-974ca344-baa9-468b-ad61-876b2a90bd7b.png">
<img width="122" alt="Screen Shot 2021-11-16 at 2 10 18 PM" src="https://user-images.githubusercontent.com/69633/142074202-a57fadf1-2b85-4b74-b5c2-0349b01b7de7.png">

When searching the Noun Project for "offline," I notice that a common visual is a cloud with a slash through it. Have we missed an opportunity to use a cloud where it makes symbolic sense? 🤔

Reply:
I like the idea of using a cloud. What do you think of this @tylersticka?

![Screen Shot 2021-11-17 at 8 47 20 AM](https://user-images.githubusercontent.com/42841342/142244509-2fae8523-db83-4bb1-959b-5e07754f52e0.png)

Reply:
I like that better. Is it possible to make the cloud larger in relation to the slash?

Reply:
It's subtle, but here's an update version with the cloud a little larger in proportion to the slash.

![Screen Shot 2021-11-17 at 9 42 12 AM](https://user-images.githubusercontent.com/42841342/142254070-d408866f-e6ec-4c1a-b7d2-76636525a2c9.png)

Reply:
I like that better!

Can the icon itself be larger within its container? I think I'm reacting to its size relative to the capital "Y" in "You"

Reply:
Here it is a little bigger:

![Screen Shot 2021-11-17 at 10 25 03 AM](https://user-images.githubusercontent.com/42841342/142260715-a101ea8a-f6bf-4b7e-993d-08f0acfe4b8d.png)

Reply:
I like that! How do you feel about it?

Reply:
I like it too!